### ISSUE TITLE
rsync-bpc fails on arm

### ISSUE THREAD

**Issue by D-sha** [backuppc/backuppc#185]
I am attempting to build backuppc on my raspberry pi 2.
I have followed the compiling instructions 
in folder rsync-bpc:
`fakeroot dpkg-buildpackage -uc -us`
fails with:

> dpkg-gencontrol: error: current host architecture 'armhf' does not appear in package's architecture list (amd64) debian/rules:87: recipe for target 'binary-arch' failed
> make: *** [binary-arch] Error 255
> dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2

I cant determine where it is getting amd64 from? 
Does anyone have a fix?
Thanks

**Comment by craigbarratt** [comment]
This should be posted on https://github.com/backuppc/rsync-bpc/issues.  

I'm not familiar with the package instructions you are trying to use.  Have you tried building and installing from the [rsync-bpc release tarball](https://github.com/backuppc/rsync-bpc/releases)?

**Comment by D-sha** [comment]
I'm sorry, I have raised this issue in the wrong place completely.
I was following the instructions on the [wiki](https://github.com/backuppc/backuppc/wiki/Build-Your-Own-Packages). I have realised this is not the official install guide and not the official github repo.
I have compiled all three parts of backuppc from tar balls now, that worked flawlessley.
I will re-raise the issue on the right repo now

Now to get apache working properley.....

Thanks for your time
